Sources/AppStoreConnectCLI/Arguments/BuildLocalizationInputArguments.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+import FileSystem
+import Foundation
+
+struct BuildLocalizationInputArguments: ParsableArguments {
+    @Argument(help: "The locale information of the build localization resource. eg. (en-AU)")
+    var locale: String
+
+    @Option(
+        parsing: SingleValueParsingStrategy.scanningForValue,
+        help: """
+            A field that describes changes and additions to a build and indicates features you would like your users to test.
+            Please put the text inside an ''.
+            eg. ('Hello There')
+        """
+    )
+    var whatsNew: String?
+
+    @Option(help: "File path to a txt file that contains whats new text. (eg. whatsNew.txt)")
+    var path: String?
+}
+
+protocol CreateUpdateBuildLocalizationCommand {
+    var localization: BuildLocalizationInputArguments { get }
+}
+
+extension CreateUpdateBuildLocalizationCommand {
+
+    func validateWhatsNewInput() throws {
+        if localization.whatsNew != nil && localization.path != nil {
+            throw ValidationError("Please provide either a file path or whats new text in terminal.")
+        }
+    }
+
+    var whatsNew: String {
+        let whatsNew: String
+
+        if let whatsNewFromOption = localization.whatsNew {
+            whatsNew = whatsNewFromOption
+        } else if let filePath = localization.path {
+            guard let whatsNewString = try? String(contentsOfFile: filePath) else {
+                fatalError("Unable to read text from file path")
+            }
+
+            whatsNew = whatsNewString
+        } else {
+            print("Reading text from stdin. (press CTRL+D to finish)")
```

Comment:
This is pretty common practice and doesn't need a log statement. However we should document that we read from `stdin` when a file or what's new isn't specified in the help text

Reply:
@adamjcampbell is abstract a good place for documenting this?
```
abstract: "Create localized What’s New text for a build. Text from `stdin` will be read when a file or what's new isn't specified."
```

Reply:
Maybe `discussion` is more suited in that same constructor. Reading the docs `abstract` is meant to be the one line description